projects/plugins/jetpack/_inc/client/at-a-glance/backups.jsx

```diff
@@ -334,39 +373,6 @@ class DashBackups extends Component {
 						) }
 					</React.Fragment>
 				);
-			case 'active': {
```

Comment:
Here I'm just moving this into a `if` statement to also consider if `preflightStatus` is `success`.